typescript/amazon-mq-rabbitmq-lambda/lib/amazon-mq-rabbitmq-lambda-stack.ts

```diff
@@ -0,0 +1,70 @@
+import { Function, Runtime, Code } from 'aws-cdk-lib/aws-lambda';
+import { InstanceClass, InstanceSize, InstanceType } from 'aws-cdk-lib/aws-ec2';
+import { RabbitMqBrokerEngineVersion, RabbitMqBrokerInstance, RabbitMqEventSource } from '@cdklabs/cdk-amazonmq';
+
+export class AmazonMqRabbitmqLambdaStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    // Define the admin secret in AWS Secrets Manager
+    const adminSecret = new secretsmanager.Secret(this, 'AdminSecret', {
+      secretName: 'AdminCredentials',
+      generateSecretString: {
+        secretStringTemplate: JSON.stringify({
+          username: 'admin' // Set a default username for RabbitMQ broker
+        }),
+        generateStringKey: 'password', // Auto-generate password
+        excludePunctuation: true,      // Avoid punctuation in password
+        passwordLength: 12,            // Set password length to 12 characters
+      },
+      removalPolicy: RemovalPolicy.DESTROY, // Ensure the secret is deleted on stack destroy
+    });
+
+    // Create a RabbitMQ broker instance with specified version and instance type
+    const broker = new RabbitMqBrokerInstance(this, 'RabbitMqBroker', {
+      publiclyAccessible: true,                 // Publicly accessible RabbitMQ broker
+      version: RabbitMqBrokerEngineVersion.V3_13, // Use RabbitMQ version 3.13
+      instanceType: InstanceType.of(InstanceClass.T3, InstanceSize.MICRO), // Instance type T3.micro
+      admin: {
+        username: adminSecret.secretValueFromJson('username').unsafeUnwrap(), // Use username from Secrets Manager
+        password: adminSecret.secretValueFromJson('password'),               // Use password from Secrets Manager
+      },
+      autoMinorVersionUpgrade: true, // Enable auto minor version upgrades
+    });
+
+    // Output the AMQP and Web Console endpoints and ports for the RabbitMQ broker
+    new CfnOutput(this, 'AmqpEndpointUrl', { value: broker.endpoints.amqp.url });
+    new CfnOutput(this, 'AmqpEndpointPort', { value: broker.endpoints.amqp.port.toString() });
+    new CfnOutput(this, 'WebConsoleUrl', { value: broker.endpoints.console.url });
+    new CfnOutput(this, 'WebConsolePort', { value: broker.endpoints.console.port.toString() });
+
+    // Create a custom CloudWatch Log Group for the consumer Lambda function
+    const consumerLambdaLogGroup = new logs.LogGroup(this, 'ConsumerLambdaLogGroup', {
+      logGroupName: 'customLogGroup', // Custom log group name
+      removalPolicy: RemovalPolicy.DESTROY, // Ensure it's deleted on stack destroy
+    });
+
+    // Define the consumer Lambda function which will handle messages from RabbitMQ
+    const consumer_lambda = new Function(this, 'consumer_lambdaFunction', {
+      runtime: Runtime.NODEJS_20_X,       // Use Node.js 20.x runtime for the Lambda function
+      code: Code.fromAsset('lambda'),     // Path to Lambda function code directory
```

Comment:
Looks like you forgot to add your lambda folder with the code.

Reply:
I've added the missing lambda folder in a new commit. Thanks for pointing it out!